src/main/resources/assets/autorun/lang/en_us.json

```diff
@@ -1,5 +1,7 @@
 {
   "key.autorun.toggle": "Toggle Auto-Run",
+  "config.autorun.toggleRunOnStart": "Toggle Run On Start",
```

Comment:
This is a bit misleading, since it doesn't toggle Auto-Run on start, but rather persists the toggled state between worlds/servers.